otherlibs/configurator/test/blackbox-tests/pkg-config-args.t/run.t

```diff
@@ -1,34 +1,40 @@
 These tests show that setting `PKG_CONFIG_ARGN` passes extra args to `pkg-config`
 
-  $ dune build 2>&1 | awk '/run:.*bin\/pkg-config/{a=1}/stderr/{a=0}a'
-  run: $TESTCASE_ROOT/_build/default/.bin/pkg-config --print-errors dummy-pkg
+  $ dune build 2>&1 | awk '/run:.*bin\/pkgconf/{a=1}/stderr/{a=0}a' | sed s/$(ocamlc -config | sed -n "/^target:/ {s/target: //; p; }")/\$TARGET/g
```

Comment:
shouldn't this rather be added to the `BUILD_PATH_PREFIX_MAP` variable at the top of this file?

Reply:
check some of the other tests for how that's done. It feels a bit cleaner to me to do it that way

Reply:
How ? Please help! :-)
This was my first idea of course. I never managed to make it work. That's why I fold back to a sed command...
After your comment, I said to myself to not be lazy and to lead this battle but I'm defeated again and again :-/
I tried all the variant I could imagine of
```
-  $ dune build 2>&1 | awk '/run:.*bin\/pkgconf/{a=1}/stderr/{a=0}a' | sed s/$(ocamlc -config | sed -n "/^target:/ {s/target: //; p; }")/\$TARGET/g
+  $ target=$(ocamlc -config | sed -n "/^target:/ {s/target: //; p; }")
+  $ printf $target
+  x86_64-pc-linux-gnu
+  $ export BUILD_PATH_PREFIX_MAP="$BUILD_PATH_PREFIX_MAP:TARGET=$target"
```
the outcome keeps being
```
+  $ dune build 2>&1 | awk '/run:.*bin\/pkgconf/{a=1}/stderr/{a=0}a'
   run: $TESTCASE_ROOT/_build/default/.bin/pkgconf --print-errors dummy-pkg
   -> process exited with code 0
   -> stdout:
    | dummy-pkg
-  run: $TESTCASE_ROOT/_build/default/.bin/pkgconf --personality $TARGET --cflags dummy-pkg
+  run: $TESTCASE_ROOT/_build/default/.bin/pkgconf --personality x86_64-pc-linux-gnu --cflags dummy-pkg
   -> process exited with code 0
   -> stdout:
    | --personality
-   | $TARGET
+   | x86_64-pc-linux-gnu
    | --cflags
    | dummy-pkg
-  run: $TESTCASE_ROOT/_build/default/.bin/pkgconf --personality $TARGET --libs dummy-pkg
+  run: $TESTCASE_ROOT/_build/default/.bin/pkgconf --personality x86_64-pc-linux-gnu --libs dummy-pkg
   -> process exited with code 0
   -> stdout:
    | --personality
-   | $TARGET
+   | x86_64-pc-linux-gnu
    | --libs
    | dummy-pkg
 ```